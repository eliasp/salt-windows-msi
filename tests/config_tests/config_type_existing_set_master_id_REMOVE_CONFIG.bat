msiexec /i test.msi START_MINION="" MASTER=master.rr4 MINION_ID=minion_id.rr4 REMOVE_CONFIG=1 /l*v config_type_existing_set_master_id_REMOVE_CONFIG.install.log /qb
