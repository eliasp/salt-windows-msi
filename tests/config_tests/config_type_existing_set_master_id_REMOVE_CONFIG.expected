# This is the template from the msi store line 1/6
master: master.rr4
# This is the template from the msi store line 2/6
id: minion_id.rr4
# This is the template from the msi store line 3/6
# This is the template from the msi store line 4/6
# This is the template from the msi store line 5/6
# This is the template from the msi store line 6/6
